fwb/app/(auth)/(routes)/detail/detail.tsx

```diff
@@ -213,8 +213,8 @@ function DetailPageContent({ data }: { data: DetailData }) {
         {/* discount listing section */}
         <div className="relative mb-[50px] xs-max:mt-[30px] xs-max:flex xs-max:flex-col xs-max:gap-[40px] xs-max:gap-[40px] xxs-max:mt-[30px] xxs-max:flex xxs-max:flex-col xxs-max:gap-[25px]">
           {(discounts.length != 0 ? discounts : data.discounts)?.map(
-            (item: DiscountDataDetail) => (
-              <DetailCard data={item} key={item.discount_amount} />
+            (item: any, index: number) => (
+              <DetailCard data={item} key={index} />
```

Comment:
Definitely way better than the discount_amount being the key 😭
Once we type this properly though, it'll prob be better to have the item's id. Not important for right now.